frontend/mulighetsrommet-veileder-flate/src/components/historikk/HistorikkButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button } from '@navikt/ds-react';
+import { useState } from 'react';
+import { useFeatureToggles, VIS_HISTORIKK } from '../../core/api/feature-toggles';
+import { useHentHistorikk } from '../../core/api/queries/useHentHistorikk';
+import StandardModal from '../modal/StandardModal';
+import { HistorikkForBruker } from './HistorikkForBruker';
+import './HistorikkForBruker.less';
+
+export function HistorikkButton() {
+  const [apneModal, setApneModal] = useState(false);
+  const features = useFeatureToggles();
+  const toggleModal = () => setApneModal(!apneModal);
+
+  const visHistorikkKnapp = features.isSuccess && features.data[VIS_HISTORIKK];
+  useHentHistorikk(visHistorikkKnapp);
```

Comment:
Dersom vi har skrudd på vis historikk så pre-fetcher vi data om historikk slik at det føles snappy og raskt for bruker når de trykker på vis historikk-knappen.